string modified_message
---
bool success